Headlines/src/Categories/UIImageEffects.swift

```diff
@@ -119,19 +119,19 @@ public extension UIImage {
             var red: CGFloat = 0
             var green: CGFloat = 0
             var blue: CGFloat = 0
-            
+
             if tintColor.getRed(&red, green: &green, blue: &blue, alpha: nil) {
                 effectColor = UIColor(red: red, green: green, blue: blue, alpha: effectColorAlpha)
             }
         }
-        
+
         return applyBlurWithRadius(10, tintColor: effectColor, saturationDeltaFactor: -1.0, maskImage: nil)
     }
-    
-    public func applyBlurWithRadius(_ blurRadius: CGFloat,
-                                    tintColor: UIColor?,
-                                    saturationDeltaFactor: CGFloat,
-                                    maskImage: UIImage? = nil) -> UIImage? {
+
+    func applyBlurWithRadius(_ blurRadius: CGFloat,
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/Canillitapp/headlines-iOS/blob/f5a5ab936ee0d164c160af7374b637afef6eb821/Headlines/src/Categories/UIImageEffects.swift#L131"></span>Function should have complexity 10 or less: currently complexity equals 17<br />
<code>cyclomatic_complexity</code> <code>UIImageEffects.swift:131</code></td>
    </tr>
  </tbody>
</table>